engine/src/vs_logging.h

```diff
@@ -13,27 +15,34 @@
  *
  * Vega Strike is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
  * along with Vega Strike. If not, see <https://www.gnu.org/licenses/>.
  */
+
+
 #ifndef VEGA_STRIKE_ENGINE_VS_LOGGING_H
 #define VEGA_STRIKE_ENGINE_VS_LOGGING_H
 
 #include <cstdint>
 
-#include <boost/move/utility_core.hpp>
-#include <boost/smart_ptr/shared_ptr.hpp>
-#include <boost/format.hpp>
-//#include <boost/log/sources/logger.hpp>
-#include <boost/log/sources/severity_logger.hpp>
-#include <boost/log/sources/record_ostream.hpp>
-#include <boost/log/sinks/sync_frontend.hpp>
-#include <boost/log/sinks/text_file_backend.hpp>
-#include <boost/log/sinks/text_ostream_backend.hpp>
-#include <boost/filesystem.hpp>
+#include "boost/move/utility_core.hpp"
```

Comment:
headers should be in `<>` enclosures not `""` (double quote) enclosures.

Reply:
Hmm. Really? That didn't compile correctly in the case of at least one source file.

Reply:
difference between them is that `<>` is stricter so the headers have to be in the path; where `""` is a little more liberal and includes the project paths and local directory. It's better to use the stricter set and configure the compiler properly to ensure the right sets of headers are easily found.

Reply:
OK. I'll see if I can get that to work.

Reply:
Seems to be working that way, with angle brackets instead of quotes. Resolving.